include/wx/osx/nonownedwnd.h

```diff
@@ -116,6 +116,8 @@ class WXDLLIMPEXP_CORE wxNonOwnedWindow : public wxNonOwnedWindowBase
 
     void OSXHandleMiniaturize(double WXUNUSED(timestampsec), bool miniaturized);
 
+	void SetIgnoreResizing(bool value) { m_ignoreResizing = value;  }
```

Comment:
This should have `OSX` prefix to follow our convention for "private" methods in public classes.